configure.ac

```
@@ -479,6 +479,11 @@ AS_IF([   test -n "$TAR"],
       [have_pack_tar=yes],
       [have_pack_tar=no])
 
+AS_IF([   test $have_pack_tar = yes \
+       && test $have_any_builder = yes],
```

Switch the order so it matches the report:

```suggestion
AS_IF([   test $have_any_builder = yes \
       && test $have_pack_tar = yes],
```